### DISCUSSION TITLE
Set home-manager targets in NixOS configuration

### DISCUSSION THREAD

**Discussion by DZZc0rd** [nix-community/stylix#634] [Q&A]
I'm trying to use stylix with hyprpaper, but without home-manager, only NixOS configuration and flakes. So, how can I enable hyprpaper target from configuration file?

**Comment by trueNAHO** [comment]
Stylix [currently only has a Home Manager module for hyprpaper](https://github.com/danth/stylix/tree/cf5be812bdc889f10ada644e4736138c5757e1e9/modules/hyprpaper). The best solution would be to submit a PR with a new `/stylix/modules/hyprpaper/nixos.nix` module.
